lib/kuniri/state_machine/OO_structured_fsm/method_state.rb

```diff
@@ -7,10 +7,21 @@ module OOStructuredFSM
     # Class responsible for handling Method state.
     class MethodState < OOStructuredState
 
+      CLASS_TYPE = "method"
+      METHOD_TYPE = "function"
+
       @language
 
-      def initialize(pLanguage)
+      @state
+      @classType
+      @methodType
+
+      def initialize(pLanguage, params = {})
```

Comment:
change: params to pParams
Add:
* Add a small documentation about each parameter (use yardoc).
* Add a unit test for check those parameters.